### PR TITLE
LRQA-56832 | master

### PR DESCRIPTION
**Pull request by vicnate5**
Relevant test.properties branch

### CODE REVIEW

**Comment by vicnate5** [comment]
ci:test:relevant

**Comment by vicnate5** [comment]
ci:test:relevant

File: modules/apps/frontend-taglib/test.properties [hunk 1]
```diff
@@ -14,5 +14,8 @@
...
 
     test.batch.run.property.query[functional-tomcat*-mysql*-jdk8][relevant]=\
-        (testray.component.names ~ "WYSIWYG") OR \
-        (testray.main.component.name ~ "User Interface")
\ No newline at end of file
+        (portal.upstream == true) AND \
+        (\
+            (testray.component.names ~ "WYSIWYG") OR \
+            (testray.main.component.name ~ "User Interface")\
+        (
```
Comment (michaelhashimoto) [inline modules/apps/frontend-taglib/test.properties]: I found it!

File: modules/apps/document-library-opener/test.properties [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    #
+    # Relevant
+    #
+
+    test.batch.run.property.query[functional-tomcat*-mysql*-jdk8][relevant]=\
+        (portal.upstream == true) AND \
+        (\
+            (testray.main.component.name ~ "Online Editing")\
+        (
```
Comment (michaelhashimoto) [inline modules/apps/document-library-opener/test.properties]: Here's another one.

**Comment by vicnate5** [comment]
ci:stop

**Comment by vicnate5** [comment]
ci:test:relevant

**Comment by vicnate5** [comment]
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/86643

**Comment by vicnate5** [comment]
ci:stop

**Comment by vicnate5** [comment]
Stopping since this will run a huge number of tests, and this was triggered on the PR I sent to Brian.

**Pull request closed without merging**